packages/ia-components/sandbox/details/DetailsDownloadOptions.jsx

```diff
@@ -71,26 +77,31 @@ export default class DetailsDownloadOptions extends React.Component {
             >
               {formats('format', k).downloadable}
               {' '}
-              <I18nIcon className="iconochive-download" en="download"/>
+              <I18nIcon className="iconochive-download" en="download" />
             </AnchorDownload>
+            {(typeof DwebArchive === 'undefined' || !DwebArchive.mirror || (k !== 'Epub')) ? null : (
+              <a href={`/epubreader/index.html?bookPath=/download/${this.props.identifier}/${downloadableFilesDict[k][0].metadata.name}`} title="Read online">
+                <I18nIcon className="iconochive-eye" en="Read online" />
+              </a>
+            )}
           </div>
         ))}
         <div className="show-all">
-          {(this.props.disconnected) ? null :
+          {(this.props.disconnected) ? null : (
             <div className="pull-right">
               <a className="boxy-ttl hover-badge" href={compressURL}>
-                <I18nIcon className="iconochive-download" en="download"/>
-                {' '}{filesCount}{' '}{I18nStr("Files")}
+                <I18nIcon className="iconochive-download" en="download" />
+                {' '}{filesCount}{' '}{I18nStr('Files')}
               </a>
-              <br/>
+              <br />
               <a className="boxy-ttl hover-badge" href={compressAllURL}>
-                <I18nSpan className="iconochive-download" en="download"/>
-                {originalFilesCount}{' '}{I18nStr("Original")}
+                <I18nSpan className="iconochive-download" en="download" />
+                {originalFilesCount}{' '}{I18nStr('Original')}
```

Comment:
this can be cleaned up 
`{originalFilesCount}{' '}{I18nStr('Original')}` => {`${originalFilesCount} ${I18nStr('Original')}`}

along with the instance up top. let's use the the backtick way if we are cobbling together string values from variables

Reply:
I use both, but ...  change made as requested though
*  I'm certain its less efficient when you are just appending constant and variable strings. 
*  I think its less readable i.e.  variable inside backtick-escape inside backtick inside react-escape  instead of variable inside react-escape